python/lsst/sims/maf/batches/glanceBatch.py

```diff
@@ -156,26 +164,39 @@ def glanceBatch(colmap=None, runName='opsim',
                                     seeingCol=colmap['seeingGeom'])
     sql = sqlConstraint
     bundle = metricBundles.MetricBundle(metric, slicer, sql, plotFuncs=subsetPlots,
-                                        displayDict=displayDict, stackerList=stackerList)
+                                        displayDict=displayDict, stackerList=stackerList,
+                                        plotDict=plotDict)
     bundleList.append(bundle)
     displayDict['caption'] = r'Proper motion precision of an $r=20$ flat SED star'
     metric = metrics.ProperMotionMetric(m5Col=colmap['fiveSigmaDepth'],
                                         mjdCol=colmap['mjd'],
                                         filterCol=colmap['filter'],
                                         seeingCol=colmap['seeingGeom'])
     bundle = metricBundles.MetricBundle(metric, slicer, sql, plotFuncs=subsetPlots,
-                                        displayDict=displayDict)
+                                        displayDict=displayDict, plotDict=plotDict)
     bundleList.append(bundle)
 
     # Solar system stuff
     displayDict['caption'] = 'Fraction of observations that are in pairs'
     displayDict['subgroup'] = 'Solar System'
+
     sql = '%s (filter="g" or filter="r" or filter="i")' % sqlC
+    pairSlicer = slicers.HealpixSlicer(nside=pairnside, latCol=colmap['dec'], lonCol=colmap['ra'])
```

Comment:
Does `pairSlicer` also need to have `latLonDeg=colmap['raDecDeg']` set when `slicers.HealpixSlicer` is called?